components/multimedia/dav1d/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 (c) Tim Mooney. All rights reserved.
+#
+BUILD_BITS=		32_and_64
+BUILD_STYLE=		meson
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		dav1d
+COMPONENT_VERSION=	0.9.2
+COMPONENT_FMRI=		codec/dav1d
+COMPONENT_SUMMARY=	A fast, cross-platform AV1 decoder
+COMPONENT_CLASSIFICATION=System/Multimedia Libraries
+COMPONENT_PROJECT_URL=	https://code.videolan.org/videolan/dav1d
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL= \
+	$(COMPONENT_PROJECT_URL)/-/archive/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:59a5fc9cc5d8ea780ad71ede6d589ed33fb5179d87780dcf80a00ee854952935
+COMPONENT_LICENSE=	BSD 2-Clause Simplified License
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
+
```

Comment:
You don't need to set COMPONENT_LICENSE_FILE. It's the default in shared-macros.mk

Reply:
Fixed.  Removed `COMPONENT_LICENSE_FILE`